docs/users/installation.md

```diff
@@ -3,69 +3,95 @@ id: installation
 title: Installation
 ---
 
-To run scalafix on your project, you must first install the Scalafix sbt plugin
-or command line interface. Currently, Scalafix does not provide any IDE
-integrations.
+First, make sure you are using a supported Scala compiler version.
 
-| Scalafix Version | Scalameta(SemanticDB) Version | Scala Version           |
-| ---------------- | ----------------------------- | ----------------------- |
-| 0.5.10           | 2.1.7                         | 2.11.12 / 2.12.4        |
-| @VERSION@        | @SCALAMETA@                   | @SCALA211@ / @SCALA212@ |
+| Scalafix  | Scala Compiler          | Scalameta   |
+| --------- | ----------------------- | ----------- |
+| 0.5.10    | 2.11.12 / 2.12.4        | 2.1.7       |
+| @VERSION@ | @SCALA211@ / @SCALA212@ | @SCALAMETA@ |
+
+Next, integrate Scalafix with your build tool or use the command-line interface.
 
 ## sbt
 
-The quickest way to get started is to run a **syntactic** rule like
-`ProcedureSyntax`
+Start by installing the sbt plugin in `project/plugins.sbt`
 
 ```scala
 // project/plugins.sbt
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "@VERSION@")
 ```
 
-From the sbt shell, invoke the `scalafix` task
+From the sbt shell, let's run a **syntactic** rule `ProcedureSyntax`
+
+```
+> myproject/scalafix ProcedureSyntax
+```
+
+If your project uses the deprecated "procedure syntax", you should have a diff
+in your sources like this
 
+```diff
+-  def unloadDrivers {
++  def unloadDrivers: Unit = {
 ```
-> myproject/scalafix ProcedureSyntax      // run rule on main sources
-> myproject/test:scalafix ProcedureSyntax // run rule on test sources
+
+Next, if we run a **semantic** rule like `RemoveUnusedImports` then we get an
+error
+
+```
+> myproject/scalafix RemoveUnusedImports
+[error] SemanticDB not found: META-INF/semanticdb/src/main/scala/...
```

Comment:
Can we put the sbt instruction in the error message?

Reply:
I would link to the docs, but I agree with the spirit of this comment!

Reply:
Let's discuss this in https://github.com/scalacenter/scalafix/issues/847 I agree with you this is not great. Good thing about documenting the UX is you see what can be improved :) Will update the docs once the situation improves.